everestrs/everestrs_sys/everestrs_sys.cpp

```diff
@@ -80,11 +81,56 @@ JsonBlob Module::call_command(rust::Str implementation_id, rust::Str name, JsonB
     return json2blob(return_value);
 }
 
+void Module::publish_variable(rust::Str implementation_id, rust::Str name, JsonBlob blob) const {
```

Comment:
why was this moved here?

Reply:
To keep all impls for Module's functions together